fem/plinearform.hpp

```diff
@@ -114,9 +114,9 @@ class ParLinearForm : public LinearForm
    void MakeRef(ParFiniteElementSpace *pf, Vector &v, int v_offset);
 
    /// Assembles the ParLinearForm i.e. sums over all domain/bdr integrators.
-   /// When @a use_device is set to true and all the ParLinearForm assembly is
+   /// When UseFastAssembly(true) has been called and the linearform assembly is
    /// compatible with device execution, it will be executed on the device.
```

Comment:
```c++
   /// Assembles the ParLinearForm i.e. sums over all domain/bdr integrators.
   /** When @ref LinearForm::UseFastAssembly "UseFastAssembly(true)" has been
       called and the linear form assembly is compatible with device execution,
       the assembly will be executed on the device. */
```

Reply:
Fixed in https://github.com/mfem/mfem/pull/3235/commits/343e7ca7c8c5f4b432f97f8632702f8ce5ef2c6c